modules/volunteers/admin/sitesupport/editHours.js

```diff
@@ -0,0 +1,116 @@
+/* jshint esversion: 6 */
+/* globals apiRequest, confirmbox, showSidebar, simpleObjectToRequest */
+
+export default {
+  data() {
+    return {
+      record: {
+        hours: 0,
+        member: {
+          first_name: '',
+        },
+        enterer: {
+          id: '',
+        },
+        authorizer: {
+          id: '',
+        },
+        department: {
+          name: '',
+        },
+        end: null,
+      },
+    }
+  },
+  methods: {
+    commitHours() {
+      confirmbox(
+        'Please! double check entries',
+        'Proceed with Volunteer Hour Update?').then(() => {
+        var item = {};
+
+        item['hours'] = parseFloat(this.record.hours);
+        item['end'] = this.record.end;
+        var e = document.getElementById('edit_mod');
```

Comment:
I can't quite put my finger on exactly why but it seems weird that we would find a need to use `document.getElementById` while using Vue. We should be able to bind to an `<option>` tag without issue and pull the value directly without the need to do this.

Reply:
I found it - we can indeed bind to `<select>` which is what I was thinking and meant, rather than `<option>`.

According to docs it looks like we should be able to do something like:

- Define in `data`
`hour_modifier: 1`

In HTML:
```
<select class="UI-select" style="width:auto" name="TimeModifier" v-model="hour_modifier">
            <option value=0.5>Half Time - 1 hour = 0.5 hours credit</option>
            <option value=1>Normal - 1 hour = 1 hour</option>
            <option value=1.5>Time Plus Half - 1 hour = 1.5 hours credit</option>
            <option value=2>Double - 1 hour = 2 hours credit</option>
</select>
```

This should result in "Normal - 1 hour = 1 hour" being the option selected by default when the page loads. Changing the selected option should be accessible via `hour_modifier` and when we go to reference this when building `item` for our request with `this.record.hour_modifier`.

https://vuejs.org/guide/essentials/forms.html#select

Reply:
Agreed, this is a place I did not clean up as much as I should, I will do a second pass. I could just drop the english wording "Double, Time Plus Half, etc.." then it because a very easy thing to populate.

Reply:
I think having the text in the `<option>` is fine, the value will be what gets sent to the back-end anyway. Whatever will be easiest for folks to understand is ideal, here.